docs/contributing.rst

```diff
@@ -137,4 +137,37 @@ Instead, the *review-notebook-app* bot will leave a comment with a **ReviewNB**
 button on every Pull Request: this lets you leave your feedback on a representation
 of the notebook itself.
 
+Some tips for reviewing
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* Does the PR have a concise description of the proposed changes? If not, request it.
+* Have a look at the proposed changes. Do they make sense?
+* Is the proposed code clean and clear?
+* Are the proposed changes documented or commented? Methods should come with docstrings. But also documentation in the form of Markdown surrounding code is very appreciated. We prefer documentation and code with variables and method names that are verbose and read like English. As an example, we prefer the second of these two snippets:
+
+
+.. code-block:: python
+    def znl_mean(ar):
+        return ar.mean('xt_ocean')
+
+.. code-block:: python
+    def zonal_mean(dataarray):
```

Comment:
```suggestion
.. code-block:: python

    def zonal_mean(dataarray):
```

I think these also need a blank line to separate from the rST metadata (otherwise it doesn't show up: https://cosima-recipes--388.org.readthedocs.build/en/388/contributing.html)